server/finders/BookFinder.js

```diff
@@ -208,6 +208,14 @@ class BookFinder {
       }
     }
 
+    if (!books.length && !options.currentlyTryingCleaned) {
+      Logger.debug(`Book Search, no matches.. checking cleaned title and author`)
+      options.currentlyTryingCleaned = true
+      return this.search(provider, this.cleanTitleForCompares(title), this.cleanAuthorForCompares(author), isbn, asin, options)
```

Comment:
The only thing I think we should check before calling search again is whether the cleaned title or the cleaned author is different from the ones just used to search. This way we don't make an unnecessary API call.

Reply:
Good idea. Just added that in.